index.html

```diff
@@ -168,6 +168,17 @@ <h3 class="contactme">LETS CONNECT</h3>
         </div>
         
     </section> 
-    
+    <section class="contact-me">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+        <form action="https://formspree.io/f/xeqwblqg" method="post">
+            <input type="text" name="name" value="Lorem Ipsum" required maxlength="30" id="name">
+            <input type="email" name="email" value="loremipsum@gmail.com" required id="email">
```

Comment:
- [ ] Please 🙏🏻 use the *HTML* elements correctly, as I can see that your `Validation` is not working with the help of the **required** attribute but it is not working because validation always works if there is no value and fortunately I got the issue that is in place of `placeholder` you are using the `value` attribute which seems like OK visually but technically it is not correct and even misleading. Kindly replace the `value` attribute with the `placeholder` attribute so that you're *required* should do work by default.